tests/core/test_processor.py

```diff
@@ -642,9 +642,16 @@ async def test_handle_message_with_session_start(
             ],
         ),
         SlotSet(entity, slot_2[entity]),
+        ActionExecuted("utter_greet"),
```

Comment:
@Ghostvv This seems weird. Why did it work before 🤯  It works now, but can you please double check?

Reply:
your fix seems correct to me. Could it be that session start was broken somehow?

Reply:
what policy `default_processor` has?

Reply:
Augmented Memo

Reply:
with max_history=3 🤔

Reply:
could it be that the `max_history` was too short before?

Reply:
Nope, a longer `max_history` doesn't break it on master

Reply:
then on master action_listen is correct because aug memo doesn't recall anything?